src/mme-app/actionHandlers/defaultMmeProcedureActionHandlers.cpp

```diff
@@ -447,17 +442,15 @@ ActStatus ActionHandlers::default_cancel_loc_req_handler(ControlBlock& cb)
 		return ActStatus::PROCEED;
 	}
 
-	MmeDetachProcedureCtxt* prcdCtxt_p = SubsDataGroupManager::Instance()->getMmeDetachProcedureCtxt();
+	MmeDetachProcedureCtxt* prcdCtxt_p =
```

Comment:
Have we tested this ? HSS initiated detach ?

Reply:
Yes. using ignite.